### ISSUE TITLE
On submitting game redirect to page with voting instructions.

### ISSUE THREAD

**Issue by local-minimum** [ludumdare/ludumdare#1169]
It is still way to easy to feel like you're done and have completed what you came for when you submit the game. Especially the first time you are here. So I suggest a super mega clear page with a bit of an instructions video along the theme of:

> Great job! You made a game. That is really awesome. Now comes the second part: Playing, Voting and Commenting. This is just as important as the first. As making the game. This is were we learn, both from our own mistakes and successes but also from seeing how other people tackled their problems. It is also about building a community, of giving constructive feedback, becoming feedback-friends and perhaps internet friends. Game making friends. I can also promise, that even if you happened to make a game that people don't understand or don't enjoy, getting feedback is way better than getting silence. And the way to ensure feedback and in the end a chance to rank among the top 100 or even become a winner, is to start playing and keep playing others' games.

**Comment by Jezzamonn** [comment]
Nice idea. One problem is usually during submission is when the site is
under the heaviest load and we usually ask people to rate games later

**Comment by local-minimum** [comment]
@Jezzamonn True, but with a little tweak, we could ask people to come back on Tuesday to start voting on games. Still I think it is good to get the information about play/voting right there and then.

**Comment by Jezzamonn** [comment]
Yeah I still really like the idea 😊

**Comment by povrazor** [comment]
Related #1206
